crates/bevy_time/src/stopwatch.rs

```diff
@@ -79,11 +80,33 @@ impl Stopwatch {
     /// # See Also
     ///
     /// [`elapsed`](Stopwatch::elapsed) - if a `Duration` is desirable instead.
+    /// [`elapsed_secs_f64`](Stopwatch::elapsed_secs_f64) - if a `f64` is desirable instead.
     #[inline]
     pub fn elapsed_secs(&self) -> f32 {
         self.elapsed().as_secs_f32()
     }
 
+    /// Returns the elapsed time since the last [`reset`](Stopwatch::reset)
+    /// of the stopwatch, in seconds, as f64.
+    ///
+    /// # Examples
```

Comment:
I'd probably cut this doc test; I don't think it's testing interesting functionality or helpful to users when the f32 example already exists.